packages/math-input/src/components/keypad/__tests__/__snapshots__/keypad.test.tsx.snap

```diff
@@ -19,13 +19,13 @@ exports[`keypad should snapshot expanded: first render 1`] = `
             aria-disabled="false"
             aria-label="Numbers"
             aria-selected="true"
-            class="button_vr44p2-o_O-reset_152ygtm-o_O-link_13xlah4-o_O-clickable_1ncqa8p"
+            class="button_vr44p2-o_O-reset_152ygtm-o_O-link_13xlah4-o_O-focused_en8zhl-o_O-clickable_1ncqa8p"
             role="tab"
             tabindex="0"
             type="button"
           >
```

Comment:
**question**: Is the fact that these now have `focused` in the class name relevant to this PR? I realize it's a snapshot, but this seems like a behaviour change.